lib/es6/caml_format.js

```diff
@@ -404,10 +404,10 @@ function parse_format(fmt) {
         case 46 :
           f.prec = 0;
           let j = i + 1 | 0;
-          while ((function () {
+          while (((() => {
```

Comment:
With the last change, this now has more parens than before.

Reply:
fixed it